testing/cypress/e2e/search-000-prereq.cy.ts

```diff
@@ -11,18 +11,39 @@ let util = new Utilities();
 let playground = new Playground();
 let req = new Request();
 
+const cookiesToClear: string[] = [
+  'KEYCLOAK_SESSION_LEGACY',
+  'KEYCLOAK_SESSION',
+  'KEYCLOAK_REMEMBER_ME',
+  'KEYCLOAK_LOCALE',
+  'KEYCLOAK_IDENTITY_LEGACY',
+  'KEYCLOAK_IDENTITY',
+  'KC_RESTART',
+  'FORMCRED',
+  'AUTH_SESSION_ID_LEGACY',
+];
+const domain: string = Cypress.env('siteminder');
+
 describe('Create Integration Requests', () => {
   beforeEach(() => {
     cy.cleanGC();
     cy.clearAllSessionStorage();
     cy.clearAllLocalStorage();
     cy.clearAllCookies();
```

Comment:
Is the clearAllCookies function still needed with the specific cookie removal?

Reply:
Yes, I think so, because it clears cookies from the main domain (and not siteminder), the specific cookie functions I added clears the cookies for the siteminder domain, so I am just making sure I clean up everything. We found this out when I built the SSO tests, so here because I use almost the same mechanism, I have just added it. Doesn't hurt.